quilt/tools/build.py

```diff
@@ -149,10 +160,7 @@ def _generate_contents(dir_path):
                 nodename, ext = splitext_no_dot(name)
                 ext = ext.lower()
                 rel_path = os.path.relpath(path, startpath)
-                if ext in TARGET['pandas']:
-                    data = [ext, rel_path]
-                else:
-                    data = ['raw', rel_path]
+                data = dict(file=rel_path)
```

Comment:
This doesn't include the transform. Shouldn't it also have:
transform=ext? (in the dict)